shotgun_api3/lib/xmlrpclib.py

```diff
@@ -142,14 +142,15 @@
 from types import *
 import socket
 import errno
-import httplib
```

Comment:
Removed this because it seemed like an unused import (no mention of `httplib` in this file).
If it is needed due to external usage, it can be changed to something like:
```
try:
    import httplib

except ImportError:
    httplib = None  # What to do in py3?
```